src/store/flow/block.ts

```diff
@@ -80,10 +80,14 @@ export const mutations: MutationTree<IFlowsState> = {
     const block = findBlockOnActiveFlowWith(blockId, state as unknown as IContext)
     findBlockExitWith(exitId, block).semantic_label = value
   },
-  block_addExit(state, {blockId, exit}: {blockId: string, exit: IBlockExit}) {
+  block_addExit(state, {blockId, exit, append}: {blockId: string, exit: IBlockExit, append?: boolean}) {
```

Comment:
@object-required , that param is a boolean, let's follow the boolean naming convention with prefix `has`, `should`, `is`, etc
Maybe `shouldAppend` is good ?